nanome_vault/menus/VaultMenu.py

```diff
@@ -14,6 +14,9 @@
 UP_ICON_PATH = os.path.join(BASE_DIR, 'icons', 'up.png')
 LOCK_ICON_PATH = os.path.join(BASE_DIR, 'icons', 'lock.png')
 
+ACCOUNT_FOLDER = 'account'
+ORG_FOLDER = 'my org'
```

Comment:
Will the space in the folder name cause complications?

Reply:
Nah, this is just what's displayed to the user, internally the path is "/org-1/" (where 1 is the org ID)